bentoml/server/bento_sagemaker_server.py

```diff
@@ -61,4 +62,5 @@ def __init__(self, bento_service, api_name, app_name=None):
         setup_routes(self.app, self.bento_service, api_name)
 
     def start(self):
+        track_server('sagemaker')
```

Comment:
notice the typo in comment above, "..an AWS sagemaker compatibility *REST* server."